src/components/ApiConnect/GlobeMotion/GlobeMotion.js

```diff
@@ -11,18 +10,15 @@ const GlobeMotion = (props) => {
         viewBox="0 0 32 32"
         className={styles.GlobeMotion}
         width={props.size}
-        height={props.size}
-      >
+        height={props.size}>
         <title>globe</title>
-        <path
-          d="M14,4a7,7,0,1,1-7,7,7,7,0,0,1,7-7m0-2a9,9,0,1,0,9,9A9,9,0,0,0,14,2Z"
-          className={styles.GlobeBall1}
-        />
-        <path d="M28,11a13.9563,13.9563,0,0,0-4.1051-9.8949L22.4813,2.5187A11.9944,11.9944,0,0,1,5.5568,19.5194l-.0381-.0381L4.1051,20.8949A13.9563,13.9563,0,0,0,14,25v3H10v2H20V28H16V24.84A14.0094,14.0094,0,0,0,28,11Z" />
-        <rect id="_Transparent_Rectangle_" fill="none" width="32" height="32" />
-      </svg>
-    </div>
+      <path className={styles.Globe} d="M14,4c3.9,0,7,3.1,7,7s-3.1,7-7,7s-7-3.1-7-7S10.1,4,14,4 M14,2c-5,0-9,4-9,9s4,9,9,9s9-4,9-9S19,2,14,2z"/>
```

Comment:
Pretty minor, but looks like we’re missing a tab here.

Reply:
This should be fixed now! I plan to add a linter in the future to catch this kind of stuff